src/Akka.Remote.Hosting/RemoteOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="AkkaRemoteOptions.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Text;
+using Akka.Configuration;
+
+namespace Akka.Remote.Hosting
+{
+    public sealed class RemoteOptions
+    {
+        public string? HostName { get; set; }
+        public int? Port { get; set; }
+        public string? PublicHostName { get; set; }
```

Comment:
I would mention that public hostname will default to `HostName` when not populated

Basically re-use what's included in the comments in the reference hocon for Akka.Remote